config/config.defaults.yml

```diff
@@ -301,14 +301,52 @@ default: &default_settings
   # development and false for other environments.
   use_js_routes_middleware: false
 
-  # Custom landing page asset host, e.g. https://s3.amazonaws.com/my-bucket/sites
-  clp_asset_host:
+  #####################
+  # Custom landing page
+  #####################
+
+  # This will enabled a static landing page
+  #
+  # The hosted version stores the landing page data in the database but for single
+  # marketplace OS installations the database doesn't bring any benifits. That's why
+  # the static version is used instead of database in OS installations.
+  clp_static_enabled: false
+
+  # Landing page is cached heavily.
+  #
+  # By default, the cache is expired after `clp_cache_time`. If you want to manually
+  # expire cache, for example after a new landing page version is deployed, you can
+  # increase this version number
+  clp_static_released_version: 1
+
+  # Path or URL for landing page assets.
+  #
+  # Must include trailing slash.
+  #
+  # You can add %{sitename} in the URL and it will be replaced with the sitename,
+  # that is saved in the landing page structure JSON.
+  #
+  # Example values:
+  #
+  # - landing_page/
+  # - https://s3.amazonaws.com/my-bucket/sites/${sitename}/
+  #
+  clp_asset_url: "landing_page/"
 
   # Default caching time for custom landing pages in seconds
   clp_cache_time: 900
 
-  # Root url for hosted proximanovasoft font files
-  font_proximanovasoft_url:
+  # Path or URL for Proxima Nova Soft font files
+  #
+  # Must include trailing slash.
+  #
+  # Example values:
+  #
+  # - empty: don't use Proximan Nova Soft font
+  # - landing_page/
+  # - https://s3.amazonaws.com/my-bucket/fonts/
+  #
+  clp_font_proximanovasoft_url:
```

Comment:
Commented already in other file but I'd rather keep the previous config option name and not scope this to landing pages.